### ISSUE TITLE
Support Python 3.x again

### ISSUE THREAD

**Issue by striglia** [striglia/pyramid_swagger#139]
We seem to have lost support for this during the 2.0.0 refactor that included bravado-core as a dependency.

See https://github.com/striglia/pyramid_swagger/blob/master/tox.ini#L2 for where we disabled testing.

We got a bug report that running 2to3 on the repo complained about this line:
```
File "/Users/.../.virtualenvs/.../lib/python3.4/site-packages/pyramid_swagger/tween.py", line 418
raise exc_class(str(exc)), None, sys.exc_info()[2]
```

Short term, we need to remove py3x trove classifiers from setup.py.
To complete this ticket, we need to:

* fix any existing non-3-compatible issues in the repo
* re-enable py3x testing in tox.ini
* put py3x trove classifiers back into setup.py

**Comment by msander** [comment]
+1 for this

**Comment by striglia** [comment]
Resolved in #149
